File: code-fights/arcade/02-centuryFromYear/go/century.go
package main

func main() {

}

func centuryFromYear(year int) int {
	c := year / 100
	if (year % 100) != 0 {
		c++
	}
	return c
}

// --------------------------------------------------
// SHORTER SOLUTION
// --------------------------------------------------
// func centuryFromYear(year int) int {
//     return 1 + (year - 1) / 100
// }

File: code-fights/arcade/02-centuryFromYear/js/century.js
// // FIRST SOLUTION
// function centuryFromYear(year) {
//     return year % 100 !== 0 ? Math.floor(year / 100) + 1 : year / 100;
// }
//

function centuryFromYear(year) {
  return Math.ceil(year / 100);
}
